x-pack/plugins/canvas/canvas_plugin_src/renderers/dropdown_filter/index.tsx

```diff
@@ -20,20 +21,37 @@ interface Config {
    * @default []
    */
   choices: string[];
+  filterGroup: string;
 }
 
+const MATCH_ALL = '%%CANVAS_MATCH_ALL%%';
+
+const getFilterValue = (filterExpression: string) => {
+  if (filterExpression === '') {
+    return MATCH_ALL;
+  }
+
+  const filterAST = fromExpression(filterExpression);
+  return get(filterAST, 'chain[0].arguments.value[0]', MATCH_ALL) as string;
```

Comment:
This seems really brittle... `get` has its usefulness, but this *appears* to have a number of runtime error vulnerabilities.